test/hocr-check/test-hocr-check.tsht

```diff
@@ -2,5 +2,7 @@
 
 plan 2
 
-file_exists sample.hocr
-exec_ok hocr-check sample.hocr
+TESTDATA="../legacy"
+
+exec_ok hocr-check "$TESTDATA/sample.html"
+cat "$TESTDATA/sample.html" | hocr-check && ok "Check from stdin"
```

Comment:
I wanted to make sure that relative paths work correctly  when changing how HTML is loaded and `-b` is picked up for `hocr-extract-images`. `legacy` were those tests that existed before #10 (see https://github.com/tmbdev/hocr-tools/pull/10#discussion_r63800429). Could be renamed `testdata`, maybe?